core/domain/subtopic_page_domain.py

```diff
@@ -348,18 +415,22 @@ def update_page_contents_audio(self, new_page_contents_audio):
         self.page_contents.recorded_voiceovers = new_page_contents_audio
 
     def update_page_contents_written_translations(
-            self, new_page_written_translations_dict):
+            self,
+            new_page_written_translations_dict: Dict[
+                str, Dict[str, Dict[str, Dict[str, object]]]
+            ]
```

Comment:
We already use this above in `SubtopicPageContentsDict`, can you create a type variable for this?

Reply:
Done!